component/Page/Profile/index.tsx

```diff
@@ -152,8 +153,8 @@ export default function ProfilePage(props: PageProps) {
                 <title>{`${data?.username || ''}'s profile`}</title>
             </Helmet>
             <LayoutContainer backgroundImage="/image/background-profile.svg">
-                <Header />
-                <div className="flex flex-1-0-0 w-full justify-center items-center z-50">
+                <Header fixedPosition={false}/>
+                <div className="flex flex-1-0-0 w-full justify-center z-50">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `fixedPosition={false}` prop to the `Header` component and the removal of `items-center` from the div class list are significant changes. Ensure these modifications do not disrupt the layout or functionality of the Header component or the div.
```diff
- <Header fixedPosition={false}/>
- <div className="flex flex-1-0-0 w-full justify-center z-50">
+ <Header />
+ <div className="flex flex-1-0-0 w-full justify-center items-center z-50">
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->